common/org.testeditor.dsl.common.ui/src/org/testeditor/dsl/common/ui/wizards/NewProjectWizard.xtend

```diff
@@ -60,6 +64,10 @@ class NewProjectWizard extends BasicNewProjectResourceWizard {
 				setRawClasspath(rawPath, null)
 			]
 		newProject.addNature(XtextProjectHelper.NATURE_ID)
+		// make sure that no target folder is included into any resource set
+		newProject.createFilter(IResourceFilterDescription.EXCLUDE_ALL.bitwiseOr(IResourceFilterDescription.FOLDERS),
+			new FileInfoMatcherDescription("org.eclipse.core.resources.regexFilterMatcher", "target"),
```

Comment:
We support maven and gradle. I think gradle uses a folder named build. if so we should exclude it alsow.